### PR TITLE
[Hub Generated] Review request for Microsoft.Kusto to add version stable/2019-09-07

### PR DESCRIPTION
**Pull request by saguiitay**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/saguiitay/azure-rest-api-specs/dev-azure-kusto-Microsoft.Kusto-2019-09-07-Public...master/).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by saguiitay** [comment]
Swagger was already approved in Private repository;

- https://portal.azure-devex-tools.com/branch/saguiitay/azure-rest-api-specs-pr/dev-azure-kusto-Microsoft.Kusto-2019-09-07...master/
- https://github.com/Azure/azure-rest-api-specs-pr/pull/900

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

File: specification/azure-kusto/resource-manager/readme.md [hunk 1]
```diff
@@ -220,17 +253,40 @@ require: $(this-folder)/../../../profiles/readme.md
...
 uncomment the  `exclude-file` section below and add the file paths.
 
 ``` yaml $(tag) == 'all-api-versions'
 #exclude-file: 
 #  - $(this-folder)/Microsoft.Example/stable/2010-01-01/somefile.json
-```
\ No newline at end of file
+```
+
+## Suppression
```
Comment (lirenhe) [inline specification/azure-kusto/resource-manager/readme.md]: duplicate suppression section.

**Review by lirenhe — APPROVED** [review]

File: specification/azure-kusto/resource-manager/readme.go.md [hunk 2]
```diff
@@ -27,11 +28,29 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
...
+output-folder: $(go-sdk-folder)/services/stable/$(namespace)/mgmt/2019-05-15/$(namespace)
 ```
+
+### Tag: package-2019-09-07 and go
+
+These settings apply only when `--tag=package-2019-09-07 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2019-09-07' && $(go)
+output-folder: $(go-sdk-folder)/services/stable/$(namespace)/mgmt/2019-09-07/$(namespace)
```
Comment (ArcturusZhang) [inline specification/azure-kusto/resource-manager/readme.go.md]: Please remove `stable` in this path. This is breaking CI in go sdk repo.

File: specification/azure-kusto/resource-manager/readme.go.md [hunk 3]
```diff
@@ -27,11 +28,29 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
...
+```
+
 ### Tag: package-2019-05-15 and go
 
 These settings apply only when `--tag=package-2019-05-15 --go` is specified on the command line.
 Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
 
 ``` yaml $(tag) == 'package-2019-05-15' && $(go)
-output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2019-05-15/$(namespace)
+output-folder: $(go-sdk-folder)/services/stable/$(namespace)/mgmt/2019-05-15/$(namespace)
```
Comment (ArcturusZhang) [inline specification/azure-kusto/resource-manager/readme.go.md]: Please remove `stable` in this path. This is breaking CI in go sdk repo.

File: specification/azure-kusto/resource-manager/readme.go.md [hunk 4]
```diff
@@ -27,11 +28,29 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2018-09-07-preview/$(namespace)
 ```
 
+### Tag: package-2019-01-21 and go
+
+These settings apply only when `--tag=package-2019-01-21 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2019-01-21' && $(go)
+output-folder: $(go-sdk-folder)/services/stable/$(namespace)/mgmt/2019-01-21/$(namespace)
```
Comment (ArcturusZhang) [inline specification/azure-kusto/resource-manager/readme.go.md]: Please remove `stable` in this path. This is breaking CI in go sdk repo.

**Review by ArcturusZhang — CHANGES_REQUESTED** [review]
Please change the `output-folder` in `readme.go.md` file.

**Review by ArcturusZhang — APPROVED** [review]

**Pull request merged**